### DISCUSSION TITLE
♻️ Reusable Content

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#772] [New in GitBook]
<img width="2293" alt="Reusable content" src="https://github.com/user-attachments/assets/929bb753-79e5-49fa-9ae0-79fbe0094dfd">

Streamline your documentation with Reusable Content in GitBook! Now, you can easily reuse and sync blocks of content in a space, saving you time and effort.

🔍 How does it work?

- **Choose and reuse**: Select the content you want to reuse and add it to other pages through the Insert menu.
- **Automatic syncing**: When you update the original content, changes automatically sync across all its instances, ensuring your docs stay up-to-date everywhere.
- **Easy management**: Manage all reusable content in your space by switching tabs in the table of contents.

👉 Give it a try! Have feedback or suggestions? Join the discussion or drop your thoughts—we’d love to hear how you’re using it!

📖 https://docs.gitbook.com/reusable-content/overview

**Comment by irs-sesc1** [comment]
That is a great addition! However, it would be even better to share the blocks between spaces. We don't see any use case for sharing blocks within a single space. But for our type of documentation, it would be a time saver to have static text blocks that all users can use across their spaces.

**Reply by sregier206** [reply]
I agree, a knowledge block/snippet, or reusable content needs to be usable everywhere to be really effective. It also needs a separate section to we can track it. What access level will this be available at?

**Comment by JaniceManwiller** [comment]
It would be nicer to be able to be able to set up a separate list of reusable items - rather than having to add text to a document and then say that you want to reuse it.

**Comment by SimonGrefGK** [comment]
First of all, thank you for introducing this fantastic feature! The concept of making content reusable is brilliant and allows generic information to be displayed multiple times across documents seamlessly.

As mentioned in your documentation, you plan to extend this feature beyond individual spaces, which is currently limited on single spaces. Enhancing access across the entire organization is precisely what I need.

I had been considering automating this process myself using GitHub Actions, which would have been quite tedious. Therefore, this feature is incredibly timely and useful.

When can we expect a beta version of the Reusable Content feature to be available organization-wide?

**Reply by RalfMania** [reply]
Do we have a chance to get part of a preview or beta testing? This feature is really important for us and therefore we would be happy to test.

**Comment by krakenhaze** [comment]
I would be happy to advocate the use of this feature in our organisation if you could manage the reusable content as read-only blocks in a side panel or separate area, and then drag those read-only blocks onto pages. The whole concept of reusable content is to ensure the same content can be reused in different places. Currently, the reusable content blocks can be overwritten and so it's not as controllable as I'd like.

**Comment by AlisonMarshallGallagher** [comment]
I have tried to reuse some content, and it displays an **empty block** with a dashed outline and a link back to the block

Is this what I should expect to see? Or should I see the content where I am reusing it?

When trying to export to a PDF, the reusable content does not show.

*I'm a new user in GitBook.  I did a general search for this, but I couldn't find any other information

**Reply by aikoii** [reply]
👋 @AlisonMarshallGallagher You should see the reusable content fully rendered. Could you share a link to your space and the page where this is happening? You can also send details to support@gitbook.com
